code/modules/client/client_procs.dm

```diff
@@ -1026,17 +1026,22 @@ GLOBAL_LIST_INIT(blacklisted_builds, list(
 					winset(src, "default-[REF(key)]", "parent=default;name=[key];command=me")
 
 /client/proc/change_view(new_size)
-	if (isnull(new_size))
-		CRASH("change_view called without argument.")
-
-	view = new_size
-	mob.hud_used.screentip_text.update_view()
-	apply_clickcatcher()
-	mob.reload_fullscreen()
-	if (isliving(mob))
-		var/mob/living/M = mob
-		M.update_damage_hud()
-	attempt_auto_fit_viewport()
+	if((!prefs?.read_preference(/datum/preference/toggle/widescreen)))
+		if (isnull(new_size))
+			CRASH("change_view called without argument.")
+
+		view = new_size
+		mob.hud_used.screentip_text.update_view()
+		apply_clickcatcher()
+		mob.reload_fullscreen()
+		if (isliving(mob))
+			var/mob/living/M = mob
+			M.update_damage_hud()
+		attempt_auto_fit_viewport()
+	else
+		mob?.hud_used?.screentip_text.update_view()
+		apply_clickcatcher()
+		mob.reload_fullscreen()
```

Comment:
Use early returns

Reply:
In what way? This is the OG code nested behind a pref check.